shelvery/rds_backup.py

```diff
@@ -189,10 +187,14 @@ def get_shelvery_backups_only(self, all_snapshots, backup_tag_prefix, rds_client
         """
         all_backups = []
         marker_tag = f"{backup_tag_prefix}:{BackupResource.BACKUP_MARKER_TAG}"
+
+        #collect tags
+        tags = all_snapshots['DBSnapshots'][0]['TagList']
```

Comment:
this is only getting the tags from the fist snapshot and then using that for all snapshots.  This needs to be in the for loop where the list_tags_for_resource api call was removed

Reply:
resolved